examples/hcp-ec2-demo/output.tf

```diff
@@ -0,0 +1,16 @@
+output "client_vpc_id" {
+  value = module.vpc.vpc_id
+}
```

Comment:
I thought about not showing the vpc id, not sure what it could be good for...

Reply:
Yeah that is a good point. I am fine with removing it for now.